src/main/java/Duke.java

```diff
@@ -1,10 +1,81 @@
+import java.util.Arrays;
+import java.util.Locale;
+import java.util.Scanner;
+
+import static java.lang.Integer.parseInt;
+
 public class Duke {
     public static void main(String[] args) {
+        Scanner myObj = new Scanner(System.in);
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        System.out.println("what can i do for you");
+
+        String input = myObj.nextLine();
+        Todos[] todo = new Todos[100];
```

Comment:
You can consider using SCREAMING_SNAKE_CASE (which I just learnt the name of), such as TODO_ARRAY or something similar